classify/js/package-lock.json

```diff
@@ -0,0 +1,1792 @@
+{
+  "name": "javascript",
```

Comment:
```
+++ b/classify/js/package-lock.json
@@ -1,5 +1,5 @@
 {
-  "name": "javascript",
+  "name": "js",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
```

(otherwise this file doesn't quite work)